new-site/src/docs/elements/components/status-label/code.mdx

```diff
@@ -0,0 +1,106 @@
+---
+slug: '/elements/components/status-label/code'
+title: 'StatusLabel - Code'
+---
+
+import { StatusLabel, Link, IconCheckCircleFill } from 'hds-react';
+import Playground from '../../../../components/Playground';
+
+## Code
+
+### Code examples 
+
+#### Default
+<Playground>
+
+```jsx
+<div id="labels-no-icon">
+  <StatusLabel>Neutral</StatusLabel>
+  <StatusLabel type="info" style={{marginLeft: 'var(--spacing-s)'}}>Info</StatusLabel>
+  <StatusLabel type="success" style={{marginLeft: 'var(--spacing-s)'}}>Success</StatusLabel>
+  <StatusLabel type="alert" style={{marginLeft: 'var(--spacing-s)'}}>Alert</StatusLabel>
+  <StatusLabel type="error" style={{marginLeft: 'var(--spacing-s)'}}>Error</StatusLabel>
+</div>
+```
+
+```html
+<div id="labels-no-icon">
+  <span class="hds-status-label">
+    Neutral
+  </span>
+  <span class="hds-status-label hds-status-label--info" style={{marginLeft: 'var(--spacing-s)'}}>
+    Info
+  </span>
+  <span class="hds-status-label hds-status-label--success" style={{marginLeft: 'var(--spacing-s)'}}>
+    Success
+  </span>
+  <span class="hds-status-label hds-status-label--alert" style={{marginLeft: 'var(--spacing-s)'}}>
+    Alert
+  </span>
+  <span class="hds-status-label hds-status-label--error" style={{marginLeft: 'var(--spacing-s)'}}>
+    Error
+  </span>
+</div>
+```
+
+</Playground>
+
+#### With an icon
+<Playground>
+
+
+```jsx
+<div id="labels-with-icon" style={{marginTop: 'var(--spacing-s)'}}>
```

Comment:
Remove this `style={{marginTop: 'var(--spacing-s)'}}` to make the example align with core version